client/src/main/java/com/influxdb/client/internal/OrganizationsApiImpl.java

```diff
@@ -80,8 +81,15 @@ public Organization findOrganizationByID(@Nonnull final String orgID) {
     @Nonnull
     @Override
     public List<Organization> findOrganizations() {
+        return findOrganizations(new OrganizationsQuery());
+    }
+
+    @Nonnull
+    @Override
+    public List<Organization> findOrganizations(OrganizationsQuery query) {
```

Comment:
Add `@Nonnull` annotation and `final` modifier:

```suggestion
    public List<Organization> findOrganizations(@Nonnull final OrganizationsQuery query) {
```